src/hooks/action.test.ts

```diff
@@ -94,4 +97,53 @@ describe('Action Hooks', () => {
       expect(queryClient.getQueryData(key)).toBeFalsy();
     });
   });
+
+  describe('useAggregateActions', () => {
+    const args = {
+      itemId,
+      view: 'builder',
+      requestedSampleSize: 5,
+      type: 'update',
+      countGroupBy: ['createdDay', 'user'],
+      aggregateFunction: 'avg',
```

Comment:
Do you support a fixed number of functions ? Can you put them in an enum or a const object to make it easier to reference ? This also means that you can type the argument more strictly (avoid "string" type if only certain values are allowed).